src/intercept-logs.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/**
```

Comment:
It might be worth including a link to https://github.com/Polymer/vscode-plugin/issues/48 somewhere in the actual source to help someone who might be new to the file understand why it exists.